conftest.py

```diff
@@ -0,0 +1,241 @@
+import json
+from pathlib import Path
+
+from pytest import fixture
+
+import os, os.path
+import shutil
+from pytest import TempPathFactory
+import unittest
+import cProfile
+import pstats
+
+# ==============================================================================
+
+import crds
+from crds.core import log, utils
+from crds.core import config as crds_config
+
+# ==============================================================================
+HERE = os.path.abspath(os.path.dirname(__file__) or ".")
+CRDS_DIR = os.path.abspath(os.path.dirname(crds.__file__))
+# CRDS_TESTING_CACHE = os.environ.get("CRDS_TESTING_CACHE", "no-test-cache-defined-see-TESTING")
```

Comment:
Unless there is reason in the near future to reference the commented-out sections, feel free to remove these completely.